### ISSUE TITLE
[BUG] Installation on Almalinux 9 subscription-manager: command not found

### ISSUE THREAD

**Issue by master3395** [litespeedtech/cyberpanel#1332]
![image](https://github.com/user-attachments/assets/ada8df63-5a88-4bae-8108-9842b8dd31e3)

**Comment by fungal** [comment]
Yep, just doing a fresh install on Alma 9.4 and came across it (ran a yum update prior to the install so it may be slightly newer Alma).

"./cyberpanel.sh: line 852: subscription-manager: command not found"
It appears after selecting Watchdog (I selected Yes) so it is very early on in the install.

Not certain what ramification this will have or if it was a line of code in the script that was corrected and this line was not removed.
